webapp/channels/src/components/team_settings_modal/team_settings_modal.tsx

```diff
@@ -36,7 +38,17 @@ const TeamSettingsModal = ({onExited, canInviteUsers, focusOriginElement}: Props
         setHasChangeTabError(false);
     }, [hasChanges]);
 
-    const handleHide = useCallback(() => setShow(false), []);
+    const handleHide = useCallback(() => {
```

Comment:
Nitpick: This logic looks awfully similar to the one in the other file. 0/5 on whether it makes sense to try to unify it (for example, by creating a custom hook for it.